MANUAL.txt

```diff
@@ -3377,6 +3377,14 @@ In this example, the document will contain a citation for `item3`
 only, but the bibliography will contain entries for `item1`, `item2`, and
 `item3`.
 
+It is possible to create a bibliography with all the citations, 
+whether or not they appear in the document, by using a wildcard:
+
+    ---
+    nocite: |
```

Comment:
I'd expect it to work both with and without the single quotes (with the single quotes, you're introducing a citation within quotes, but since it's not printed it doesn't matter).

And of course the single quotes would be needed if you didn't use the `|` quoting mechanism (as was the case in the example you link to).

So, either

```
nocite: |
  @*
```
or
```
nocite: '@*'
```